### DISCUSSION TITLE
map is not fetching

### DISCUSSION THREAD

**Discussion by NivethaE14** [upyog/UPYOG#1117] [Q&A]
![Screenshot (47)](https://github.com/upyog/UPYOG/assets/166114393/43f22ac6-9b6d-454d-8cd1-98cd4e5c4139)
It is not working what should be done for this?

**Comment by vinothrallapalli-eGov** [comment]
Please refer to the document to configure maps. Once the secrets are generated, they need to be updated in the secrets under the egov-location service as well as in the globalconfig.js file.

https://upyog-docs.gitbook.io/upyog-v-2.0/upyog-1/platform/configure-upyog/master-data-collection-templates/environment-setup/state-level-setup/google-map-configuration
